src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Authentication/Handler/LogoutSuccessHandler.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright 2016 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupSelfService\SelfServiceBundle\Security\Authentication\Handler;
+
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Http\Logout\LogoutSuccessHandlerInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+final class LogoutSuccessHandler implements LogoutSuccessHandlerInterface
+{
+    /**
+     * @var TokenStorageInterface
+     */
+    private $tokenStorage;
+
+    /**
+     * @var string[]
+     */
+    private $logoutRedirectUrl;
+
+    /**
+     * @param TokenStorageInterface $tokenStorage
+     * @param string[] $locales
+     * @param string[] $logoutRedirectUrl
+     */
+    public function __construct(TokenStorageInterface $tokenStorage, array $locales, array $logoutRedirectUrl)
```

Comment:
Nice solution for this problem. Some open ends:
- The parameter `$locales` is never used. So either use them for verification and or picking the default language if the users preferred locale is not set.
- Consider adding some test coverage for this class.
- use Symfony\Component\Translation\TranslatorInterface; is not used. And can be removed from the 'imports' list.

Reply:
Removed the$locales parameter and removed the unused import. At the moment I can't justify the time for adding test coverage for the logoutsuccesshandler so I'll skip that.